react-accordion/my-app/src/Accordion.tsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from 'react';
+import { Topic } from './App';
+
+type Props = {
+  topics: Topic[];
+};
+
+export function Accordion({ topics }: Props) {
+  const [currentID, setCurrentID] = useState<number>();
+
+  const mappedTopics = topics.map((topic) => (
+    <TopicsCard
+      card={topic}
+      key={topic.id}
+      display={topic.id === currentID}
+      onTitleClick={(target) =>
```

Comment:
Interestingly, you can actually do this without a parameter:
`onTitleClick={() => topic.id === currentID ? setCurrentID(undefined) : setCurrentID(topic.id)}`
This works because you know the topic that you are currently rendering.